localization/pose_twist_fusion_filter/ekf_localizer/src/ekf_localizer.cpp

```diff
@@ -51,15 +51,13 @@ EKFLocalizer::EKFLocalizer(const std::string & node_name, const rclcpp::NodeOpti
   pose_stddev_x_ = declare_parameter("pose_stddev_x", 0.05);
   pose_stddev_y_ = declare_parameter("pose_stddev_y", 0.05);
   pose_stddev_yaw_ = declare_parameter("pose_stddev_yaw", 0.035);
-  use_pose_with_covariance_ = declare_parameter("use_pose_with_covariance", false);
 
   /* twist measurement */
   twist_additional_delay_ = declare_parameter("twist_additional_delay", 0.0);
   twist_rate_ = declare_parameter("twist_rate", 10.0);  // used for covariance calculation
   twist_gate_dist_ = declare_parameter("twist_gate_dist", 10000.0);  // Mahalanobis limit
   twist_stddev_vx_ = declare_parameter("twist_stddev_vx", 0.2);
   twist_stddev_wz_ = declare_parameter("twist_stddev_wz", 0.03);
```

Comment:
Let's remove these params too.

Reply:
@YamatoAndo OK. If user wants to adjust covariance of pose / twist, is it correct to imagine adjusting ndt_scan_matcher / vehicle_velocity_conveter side?

Reply:
yes!!

Reply:
fixed f6a1cc9c4a5ffd91e8efb02a4328a2463c3a9dbc. I will fix launcher side.